site/docs/latest/releaseNotes.md

```diff
@@ -0,0 +1,483 @@
+
+Prior to this release, Apache BookKeeper only supports simple `DIGEST-MD5` type authentication.
+
+With this release of Apache BookKeeper, a number of feature are introduced that can be used, together of separately,
+to secure a BookKeeper cluster.
+
+The following security features are currently supported.
+
+- Authentication of connections to bookies from clients, using either `TLS` or `SASL (Kerberos).
+- Authentication of connections from clients, bookies, autorecovery daemons to `ZooKeeper`, when using zookeeper
+    based ledger managers.
+- Encryption of data transferred between bookies and clients, between bookies and autorecovery daemons using `TLS`.
+
+It's worth noting that those security features are optional - non-secured clusters are supported, as well as a mix
+of authenticated, unauthenticated, encrypted and non-encrypted clients.
+
+For more details, have a look at [BookKeeper Security](../security).
+
+### Public API
+
+There are multiple new client features introduced in 4.5.0.
+
+#### LedgerHandleAdv
+
+The [Ledger API] is the low level API provides by BookKeeper for interacting with `ledgers` in a bookkeeper cluster.
+It is simple but not flexible on ledger id or entry id generation. Apache BookKeeper introduces `LedgerHandleAdv`
+as an extension of existing `LedgerHandle` for advanced usage. The new `LedgerHandleAdv` allows applications providing
+its own `ledger-id` and assigning `entry-id` on adding entries.
+
+See [Ledger Advanced API](../api/ledger-adv-api) for more details.
+
+#### Long Poll
+
+`Long Poll` is a main feature that [DistributedLog](https://distributedlog.io) uses to achieve low-latency tailing.
+This big feature has been merged back in 4.5.0 and available to BookKeeper users. It allows tailing-reads without
+polling `LastAddConfirmed` everytime after the readers exhaust known entries.
+
+Although `Long Poll` brings great latency improvements on tailing reads, it is still a very low-level primitive.
+It is still recommended to use high level API (e.g. [DistributedLog API](../api/distributedlog-api)) for tailing and streaming use cases.
+
+See [Streaming Reads](https://distributedlog.incubator.apache.org/docs/latest/user_guide/design/main.html#streaming-reads) for more details.
+
+#### Explicit LAC
+
+Prior to 4.5.0, the `LAC` is only advanced when subsequent entries are added. If there is no subsequent entries added,
+the last entry written will not be visible to readers until the ledger is closed. High-level client (e.g. DistributedLog) or applications
+has to work around this by writing some sort of `control records` to advance `LAC`.
+
+In 4.5.0, a new `explicit lac` feature is introduced to periodically advance `LAC` if there are not subsequent entries added. This feature
+can be enabled by setting `explicitLacInterval` to a positive value.
+
+### Performance
+
+There are a lot for performance related bug fixes and improvements in 4.5.0. The major performance improvement introduced in 4.5.0, is
```

Comment:
I think there are lot more perf improvements related to  caching additional metadata, avoiding buffer copies, taking advantage of java8 features, scheduling and threading improvements etc. Ledger recovery improvements. We should discuss these.